### PR TITLE
docs: Fix typo in dbtgmt/migration.rst

### PR DESCRIPTION
**Pull request by obozdag**
**Description**
Fix typo in dbtgmt/migration.rst

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [x] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

**Review by michalsn — COMMENTED** [review]
I see that in line 199 we use this type of convention for CLI options. Why do you want to remove it here?

**Comment by obozdag** [comment]
@michalsn Just to supply consistency with the other commands explanations.
You can use (``make:migration``) with the following options:
You can use (status) with the following options:
You can use (refresh) with the following options:
You can use (rollback) with the following options:
You can use (migrate) with the following options:

**Comment by michalsn** [comment]
Ok - thank you. I get it now, however, I would rather use this convention in other places too instead of removing it.

It's a cosmetic change, but let's wait for other opinions. @kenjis is more up-to-date when it comes to the conventions we use across the user guide, so I will rely on his opinion here.

File: user_guide_src/source/dbmgmt/migration.rst [hunk 1]
```diff
@@ -209,7 +209,7 @@ creates is the Pascal case version of the filename.
 
   php spark make:migration <class> [options]
 
-You can use (``make:migration``) with the following options:
+You can use (make:migration) with the following options:
```
Comment (kenjis) [inline user_guide_src/source/dbmgmt/migration.rst:212]: We use <code>``</code> for source code or CLI commands, so I think the following is better.
I don't know why there are `()` in here.

```suggestion
You can use ``make:migration`` with the following options:
```

**Comment by kenjis** [comment]
> You can use (status) with the following options:
You can use (refresh) with the following options:
You can use (rollback) with the following options:
You can use (migrate) with the following options:

I think it is better to change them to like `You can use ``migrate:status`` with the following options:` for consistency.

**Comment by kenjis** [comment]
One more thing, it is better to change the section title `rollback` to `migrate:rollback`.
https://codeigniter4.github.io/CodeIgniter4/dbmgmt/migration.html#rollback

These titles, too.
https://codeigniter4.github.io/CodeIgniter4/dbmgmt/migration.html#refresh
https://codeigniter4.github.io/CodeIgniter4/dbmgmt/migration.html#status

**Review by kenjis — APPROVED** [review]
Thank you! LGTM!

**Review by michalsn — APPROVED** [review]

**Comment by kenjis** [comment]
@obozdag Thank you!

**Pull request merged**